miso-web/src/it/java/uk/ac/bbsrc/tgac/miso/webapp/integrationtest/page/BulkDilutionPage.java

```diff
@@ -0,0 +1,86 @@
+package uk.ac.bbsrc.tgac.miso.webapp.integrationtest.page;
+
+import static org.openqa.selenium.support.ui.ExpectedConditions.*;
+
+import java.util.Collection;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+import com.google.common.base.Joiner;
+
+import uk.ac.bbsrc.tgac.miso.webapp.integrationtest.page.element.HandsOnTable;
+
+public class BulkDilutionPage extends AbstractPage {
+
+  public static class Columns {
+    public static final String NAME = "Library Name";
```

Comment:
These look to be all Library columns, not Dilution columns. If this is placeholder material/to be extended later, could you add a `TODO`?